app/pages/my-dashboard.tsx

```diff
@@ -80,6 +80,7 @@ function MyDashboard({ currentUser }: Props) {
 
   const handleEditClick = (id: string) => {
     // router.push(`/realm/${id}`);
+    window.scrollTo(0, 0);
```

Comment:
Is `window` a built in function in react? i noticed it wasn't imported?

Reply:
I believe `window` object is default browser object.